frontend/src/components/ReviewersPage.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="row">
+    <div class="col-sm-12 col-md-9">
+      <div v-for="reviewer of reviewers" class="card mb-3">
+        <div class="card-header">
+          <h5 class="card-title">{{ reviewer.memberProfile.name }}</h5>
+        </div>
+        <div class="card-body">
+          <p class="card-text">
+            <strong>Email:</strong> {{ reviewer.memberProfile.email }}<br />
+            <strong>Programming Languages:</strong> {{ reviewer.programmingLanguages.join(", ")
+            }}<br />
+            <strong>Subject Areas:</strong> {{ reviewer.subjectAreas.join(", ") }}<br />
+            {{ reviewer.notes }}
+          </p>
+          <a
+            class="btn btn-primary"
+            @click="
+              editCandidate = reviewer;
+              editFormRef?.resetForm();
+              editModal?.show();
+            "
+            >Edit</a
+          >
+          <a class="btn btn-danger" @click="deleteReviewer(reviewer)">Delete</a>
```

Comment:
for inactive reviews this can do the opposite

Reply:
So should inactive reviewers still show up in the list (maybe with (Inactive) added beside their name)?

Reply:
should be some indicator in the list but also filtered out by default with the option to un-select that filter